eclair-core/src/main/scala/fr/acinq/eclair/transactions/Transactions.scala

```diff
@@ -127,10 +127,10 @@ object Transactions {
     override val toLocalDelayedWeight = 483
     override val toRemoteWeight = 442
     override val htlcDelayedWeight = 483
-    override val mainPenaltyWeight = 484
```

Comment:
How do you get this value ? I find that for a penalty tx the base size is 82 bytes and there are 3 items on the witness stack: a signature, 0x01, and the redeem script which is 77 bytes
With a 73 bytes DER signature we get a witness size of 154 bytes and a weight of 483  bytes ?